test/login.mjs

```diff
@@ -60,12 +62,18 @@ describe("login", function () {
       getToken: stub().resolves({ accessToken: "access-token" }),
     };
   };
+
   beforeEach(() => {
+    const __filename = fileURLToPath(import.meta.url);
+    const __dirname = path.dirname(__filename);
+    const homedir = path.join(__dirname, "../../test/test-homedir");
```

Comment:
our runtime is new enough we get a slightly nicer bit of syntactic sugar:
```
const __dirname = import.meta.dirname;
```

Reply:
nice